src/Symfony/Component/HttpFoundation/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ CHANGELOG
 5.3
 ---
 
+ * Calling `Request::getSession` when there is not available session throws an `SessionNotFoundException`.
```

Comment:
`Calling Request::getSession() when there is no available session throws a SessionNotFoundException`

Reply:
done